src/main/java/pl/sda/javapoz/service/UserService.java

```diff
@@ -79,7 +84,7 @@ public User findUserById(Long id) {
         u2.setFirstName("Janina");
         u2.setLastName("Nowak");
         u2.setEmail("janina@gmail.com");
-        u2.setPassword("haslo");
+        u2.setPassword(passwordEncoder.encode("haslo"));
```

Comment:
To raczej powinno byc kodowane w momencie zapisywania usera. Więc raczej trzeba zmienić motodę w repo.

Reply:
też tak myślę :)